EIPS/eip-721.md

```diff
@@ -0,0 +1,312 @@
+
+Differences between this standard and EIP-20 are examined below.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+The **baseline specification** is REQUIRED for all ERC-721 implementations (subject to "caveats", below).
+
+```solidity
+pragma solidity ^0.4.19;
+
+/// @title Interface for contracts conforming to ERC-721: Deed Standard
+/// @author William Entriken (https://phor.net), et. al.
+/// @dev Specification at https://github.com/ethereum/eips/XXXFinalUrlXXX
+interface ERC721 {
+
+    // COMPLIANCE WITH ERC-165 (DRAFT) /////////////////////////////////////////
+
+    /// @dev ERC-165 (draft) interface signature for itself
+    // bytes4 internal constant INTERFACE_SIGNATURE_ERC165 = // 0x01ffc9a7
+    //     bytes4(keccak256('supportsInterface(bytes4)'));
+
+    /// @dev ERC-165 (draft) interface signature for ERC721
+    // bytes4 internal constant INTERFACE_SIGNATURE_ERC721 = // 0xe55729f5
+    //     bytes4(keccak256('ownerOf(uint256)')) ^
+    //     bytes4(keccak256('countOfDeeds()')) ^
+    //     bytes4(keccak256('countOfDeedsByOwner(address)')) ^
+    //     bytes4(keccak256('deedOfOwnerByIndex(uint256)')) ^
+    //     bytes4(keccak256('approve(address,uint256)')) ^
+    //     bytes4(keccak256('takeOwnership(uint256)'));
+
+    /// @notice Query a contract to see if it supports a certain interface
+    /// @dev Returns `true` the interface is supported and `false` otherwise,
+    ///  returns `true` for INTERFACE_SIGNATURE_ERC165 and
+    ///  INTERFACE_SIGNATURE_ERC721, see ERC-165 for other interface signatures.
+    function supportsInterface(bytes4 _interfaceID) external pure returns (bool);
+
+    // PUBLIC QUERY FUNCTIONS //////////////////////////////////////////////////
+
+    /// @notice Find the owner of a deed
+    /// @param _deedId The identifier for a deed we are inspecting
+    /// @dev Deeds assigned to zero address are considered destroyed, and
+    ///  queries about them do throw.
+    /// @return The non-zero address of the owner of deed `_deedId`, or `throw`
+    ///  if deed `_deedId` is not tracked by this contract
+    function ownerOf(uint256 _deedId) external view returns (address _owner);
+
+    /// @notice Count deeds tracked by this contract
+    /// @return A count of the deeds tracked by this contract, where each one of
+    ///  them has an assigned and queryable owner
+    function countOfDeeds() external view returns (uint256 _count);
+
+    /// @notice Count all deeds assigned to an owner
```

Comment:
Can one deed be assigned to multiple owners?

Reply:
I think that would need to be done through a secondary contract. In other words, a contract (intermediary) would own the logic and decision making about that particular asset.